### DISCUSSION TITLE
Action-packed June! Actions Check-in

### DISCUSSION THREAD

**Discussion by grandmas-favorite** [community/community#130640] [Actions]
![Header for the Actions category shows a dark purple abstract background with white text reading “GitHub Actions Check-in”. In the foreground there is an image of a person holding a magnifying glass next to a cartoon representation of a computer screen with a moving gear and play button. To the right, there is a GitHub logo in two-tone purple.](https://github.com/user-attachments/assets/5df16fc1-ef3c-4c82-b5d3-cadb792d8c01)

Hello Community! 👋 

It’s heating up here in the Northern Hemisphere, and the Actions category is no different! Join me in celebration all of these wonderful Community interactions that took place in the category last month:

- [vshunkov](https://github.com/vshunkov) helped out a crowd of Community members ([way0utwest](https://github.com/way0utwest), [lwasser](https://github.com/lwasser), and [idroulia](https://github.com/idroulia)) figure out [why their Jekyll builds were breaking](https://github.com/orgs/community/discussions/127006). Hours of head-scratching saved by [vshunkov](https://github.com/vshunkov)’s generosity! 🎉

- Shout out to [judahpaul16](https://github.com/judahpaul16) for helping [theinfamouscoder5](https://github.com/theinfamouscoder5) get a stuck action out of the queue with [their straightforward and thorough advice](https://github.com/orgs/community/discussions/126219). ✅

- When [Sonichigo](https://github.com/Sonichigo) couldn’t figure out [how to get the ACTIONS_ID_TOKEN_REQUEST_URL env variable](https://github.com/orgs/community/discussions/123735), they ended up figuring it out themselves. Although they could have easily gone on their own way and left the Discussion abandoned, they decided to come back and post their solution for the benefit of the rest of the Actions Community. Round of applause for [Sonichigo](https://github.com/Sonichigo)!

- 📰Looking for advice and tips to manage your Actions costs and consumption? [Read](https://github.com/orgs/community/discussions/129964) Hubber @kittychiu’s guide on the Community!

If you love open source like we do, you can always [contribute](https://github.com/github/docs/edit/main/content/index.md) directly to any doc by submitting a PR, learn more about [how to contribute](https://github.com/github/docs/blob/main/CONTRIBUTING.md) to GitHub docs. 📝 

#### [ArmⓇ-based Linux and Windows hosted runners for GitHub Actions](https://github.blog/changelog/2024-06-03-actions-arm-based-linux-and-windows-runners-are-now-in-public-beta)! 
This new addition to our suite of hosted runners provides power, performance & sustainability improvements for all your Actions jobs. Developers can now take advantage of Arm-based hardware hosted by GitHub to build and deploy their release assets anywhere Arm architecture is used. These runners are priced at 37% less than our x64 Linux and Windows runners!

Share your thoughts [here](https://github.com/orgs/community/discussions/127102) in the Community feedback thread!

#### [Artifact Attestations is generally available](https://github.blog/changelog/2024-06-25-artifact-attestations-is-generally-available)!

We’re thrilled to announce the general availability of GitHub Artifact Attestations! Artifact Attestations allow you to guarantee the integrity of artifacts built inside GitHub Actions by creating and verifying signed attestations. With this release, you can now easily verify these artifacts before you deploy them in your Kubernetes cluster. Powered by [Sigstore](https://www.sigstore.dev/), Artifact Attestations help you secure your software’s supply chain by creating an unforgeable link between artifacts and their build process.

#### Important dates

Starting November 30, 2024, GitHub Actions customers will no longer be able to use v3 of actions/upload-artifact or actions/download-artifact. Please see [the documentation](https://github.com/actions/upload-artifact/blob/main/docs/MIGRATION.md) in the project repositories for guidance on how to migrate your workflows. 

Also, as of May 15th, 2024, we no longer support multiple labels on larger GitHub Hosted Runners. More on this [here](https://github.blog/changelog/2024-04-16-deprecation-of-multi-label-larger-runners/).

### In case you missed these GitHu-pdates…

- Github Universe 2024 tickets are now on sale! As the world’s fair of software, GitHub Universe 2024 will be an unparalleled gathering of the brightest minds, companies, and innovators in the industry. With sessions diving into AI, the developer experience (DevEx), and security, attendees will have an opportunity to explore the latest products, best practices, and insights shaping the future of software development.

  -[ Get tickets 35% off here only from now until July 8!](https://github.blog/2024-04-16-the-worlds-fair-of-software-join-us-at-github-universe-2024/)

-  Show everyone that you’re a GitHub expert with our [GitHub Certifications](https://github.blog/2024-01-08-github-certifications-are-generally-available/), now generally available. What was once reserved for employees and partners is now available to anyone who wants to prove their GitHub wit! 📜

What are you most excited about for the month of July? Comment or share a reaction below!

**Comment by VadymMakohon** [comment]
Hello Community! 👋

July is already shaping up to be an exciting month, and I’m most thrilled about a couple of key updates!

Firstly, the Arm-based Linux and Windows hosted runners are a game changer. The promise of enhanced performance, sustainability, and the fact that they are 37% more cost-effective than x64 runners is huge. I can’t wait to explore how they’ll optimize workflows and improve build times, especially for teams focused on scalability and efficiency.

Secondly, the Artifact Attestations going GA is a major win for security! Ensuring artifact integrity with signed attestations really raises the bar for securing the software supply chain, and I’m looking forward to seeing how teams leverage this to build safer deployments.

And, of course, with GitHub Universe 2024 just around the corner, it’s exciting to imagine all the AI, DevEx, and security insights that will be shared at the event. Definitely on my radar!

What about you all? What’s caught your eye this month? Feel free to share below! 😊
